client/webserver/site/src/js/app.ts

```diff
@@ -811,15 +811,7 @@ export default class Application {
     const supportedAsset = this.assets[assetID]
     if (supportedAsset) return supportedAsset.info.unitinfo
     if (!xc) {
-      console.error(`no supported asset info for id = ${assetID}, and no exchange info provided`)
-      return {
-        atomicUnit: '',
-        conventional: {
-          unit: '',
-          conversionFactor: 1e8
-        },
-        denominations: []
-      }
+      throw Error(`no supported asset info for id = ${assetID}, and no exchange info provided`)
```

Comment:
Now that we actually have `conversionFactors`s that aren't `1e8`, I don't think we should return a default value here anymore.